eudemo/eudemo/reactor.py

```diff
@@ -225,8 +226,7 @@ def _read_json(file_path: str) -> Dict:
 
 
 if __name__ == "__main__":
-    import time
-
+    set_log_level("DEBUG")
```

Comment:
Personally, I think debug is too verbose as a default, even for development purposes.
I'm not normally that interested in the execution timings/all the other stuff that comes with debug (like the messages about the dyn executed modules).

Reply:
sure I'll leave it there so it can easily be changed but set it to info